kroxylicious-operator/src/main/resources/META-INF/fabric8/kafkaproxyingresses.kroxylicious.io-v1.yml

```diff
@@ -59,4 +60,51 @@ spec:
                     protocol:
                       description: "Protocol specifies the network protocol this ingress expects to receive."
                       type: string
-                      enum: ["TCP"]
\ No newline at end of file
+                      enum: ["TCP"]
+            status:
+              type: object
+              properties:
+                observedGeneration:
+                  description: |
+                    The metadata.generation that was observed during the last reconciliation by the operator.
+                  type: integer
+                conditions:
+                  # Mapped to Java type io.kroxylicious.kubernetes.api.common.Condition
+                  type: array
```

Comment:
We should use the description to describe the conditions we use:

https://github.com/kubernetes-sigs/gateway-api/blob/e68481c526a7cc0b1989914cad540f95aa6d4036/config/crd/standard/gateway.networking.k8s.io_gateways.yaml#L910

Reply:
Yes, I agree, but I don't think we need to do that in this PR. We [don't have a completely clear idea](https://github.com/kroxylicious/kroxylicious/pull/1969#discussion_r2004480137) about exactly how we're going to use which conditions right now. Moreover I suspect it might take a bit of trial and error, so I'd rather not hold this PR up.